### ISSUE TITLE
unnecessary multiple log output.

### ISSUE THREAD

**Issue by ooyamams** [ossc-db/pgaudit#8]
pgaudit.conf

```
[output]
logger= 'serverlog'

[rule]
database = 'foo'
class = 'READ'

[rule]
database = 'baa'
class = 'READ'
```

then executed these commands.

```
$psql -d foo
foo=# select 1;
```

postgresql.log

```
LOG:  AUDIT: SESSION,1,1,READ,SELECT,,select 1;,<not logged>
LOG:  AUDIT: SESSION,1,1,READ,SELECT,,select 1;,<not logged>
```

I tried to add another one rule.
pgaudit.conf

```
[output]
logger= 'serverlog'

[rule]
database = 'foo'
class = 'READ'

[rule]
database = 'baa'
class = 'READ'

[rule]
database != 'yada'
class = 'READ'
```

```
$psql -d foo
foo=# select 1;
```

then triplet were logged.

```
LOG:  AUDIT: SESSION,1,1,READ,SELECT,,select 1;,<not logged>
LOG:  AUDIT: SESSION,1,1,READ,SELECT,,select 1;,<not logged>
LOG:  AUDIT: SESSION,1,1,READ,SELECT,,select 1;,<not logged>
```

**Comment by MasahikoSawada** [comment]
Could you please try this again with current HEAD?
I should have fixed that issue by 911d6cbbef1f678f92e3a0d312168e25e9fa18ad.

**Comment by ooyamams** [comment]
Fixed by b9e94eb653f0e35eab4305cbad368413df958a7b. Close.
